### ISSUE TITLE
2080S OPENCL ERROR 

### ISSUE THREAD

**Issue by ytQiao** [filecoin-project/lotus#2732]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

**Describe the problem**

In the precommit2 phase, there are still 2080s graphics cards used for sealing, and there are still opecl errors. I think it needs to be optimized

**Lotus storage miner logs**
```
2020-07-30T23:43:13.060 INFO filcrypto::proofs::api > seal_commit_phase2: start
2020-07-30T23:43:13.221 INFO filecoin_proofs::api::seal > seal_commit_phase2:start
2020-07-30T23:43:13.221 INFO filecoin_proofs::caches > trying parameters memory cache for: STACKED[34359738368]
2020-07-30T23:43:13.221 INFO filecoin_proofs::caches > found params in memory cache for STACKED[34359738368]
2020-07-30T23:43:13.221 INFO filecoin_proofs::api::seal > got groth params (34359738368) while sealing
2020-07-30T23:43:13.221 INFO filecoin_proofs::api::seal > snark_proof:start
2020-07-30T23:43:13.344 INFO bellperson::groth16::prover > Bellperson 0.9.2 is being used!
2020-07-31T01:01:29.037 INFO bellperson::gpu::locks > GPU is available for FFT!
2020-07-31T01:01:29.177 DEBUG bellperson::gpu::locks > Acquiring GPU lock...
2020-07-31T01:01:29.177 DEBUG bellperson::gpu::locks > GPU lock acquired!
2020-07-31T01:01:44.035 INFO bellperson::gpu::fft > FFT: 1 working device(s) selected.
2020-07-31T01:01:44.036 INFO bellperson::gpu::fft > FFT: Device 0: GeForce RTX 2080 SUPER
2020-07-31T01:01:44.036 INFO bellperson::domain > GPU FFT kernel instantiated!
2020-07-31T01:01:52.609 WARN bellperson::gpu::locks > GPU FFT failed! Falling back to CPU... Error: Ocl Error: 

################################ OPENCL ERROR ############################### 

Error executing function: clEnqueueNDRangeKernel("radix_fft")  

Status error code: CL_MEM_OBJECT_ALLOCATION_FAILURE (-4)  

Please visit the following url for more information: 

https://www.khronos.org/registry/cl/sdk/1.2/docs/man/xhtml/clEnqueueNDRangeKernel.html#errors  

############################################################################# 

2020-07-31T01:03:41.371 WARN bellperson::gpu::locks > GPU FFT failed! Falling back to CPU... Error: Ocl Error: 

################################ OPENCL ERROR ############################### 

Error executing function: clEnqueueNDRangeKernel("radix_fft")  

Status error code: CL_MEM_OBJECT_ALLOCATION_FAILURE (-4)  

Please visit the following url for more information: 

https://www.khronos.org/registry/cl/sdk/1.2/docs/man/xhtml/clEnqueueNDRangeKernel.html#errors  

############################################################################# 

2020-07-31T01:04:32.459 WARN bellperson::gpu::locks > GPU FFT failed! Falling back to CPU... Error: Ocl Error: 

……

2020-07-31T02:07:07.693 DEBUG bellperson::gpu::locks > GPU lock released!
2020-07-31T02:08:17.656 INFO bellperson::gpu::locks > GPU is available for Multiexp!
2020-07-31T02:08:17.656 DEBUG bellperson::gpu::locks > Acquiring GPU lock...
2020-07-31T02:08:17.656 DEBUG bellperson::gpu::locks > GPU lock acquired!
2020-07-31T02:08:18.040 INFO bellperson::gpu::multiexp > Multiexp: 1 working device(s) selected. (CPU utilization: 0)
2020-07-31T02:08:18.040 INFO bellperson::gpu::multiexp > Multiexp: Device 0: GeForce RTX 2080 SUPER (Chunk-size: 5286352)
2020-07-31T02:08:18.040 INFO bellperson::multiexp > GPU Multiexp kernel instantiated!
2020-07-31T03:08:25.868 DEBUG bellperson::gpu::locks > GPU lock released!
2020-07-31T03:08:28.934 INFO filecoin_proofs::api::seal > snark_proof:finish
2020-07-31T03:08:28.946 INFO filecoin_proofs::api::seal > verify_seal:start
2020-07-31T03:08:28.958 INFO filecoin_proofs::caches > trying parameters memory cache for: STACKED[34359738368]-verifying-key
2020-07-31T03:08:28.959 INFO filecoin_proofs::caches > found params in memory cache for STACKED[34359738368]-verifying-key
2020-07-31T03:08:28.959 INFO filecoin_proofs::api::seal > got verifying key (34359738368) while verifying seal
2020-07-31T03:08:29.061 INFO filecoin_proofs::api::seal > verify_seal:finish
2020-07-31T03:08:29.061 INFO filecoin_proofs::api::seal > seal_commit_phase2:finish
2020-07-31T03:08:29.067 INFO filcrypto::proofs::api > seal_commit_phase2: finish

Please go through the logs of your storage miner, and include screenshots of any error-like messages you find.
```

**Version**
Daemon:  0.4.1+git.477dd536+api0.8.1
Local: lotus version 0.4.1+git.477dd536

@whyrusleeping @magik6k

**Comment by FreeBoys** [comment]
I have the same problem.

**Comment by Meatball13** [comment]
Same issue in the C2 phase, additionally, in my experience, the GPU lock is almost always released at 45 minutes into the cycle and then the GPU lock is successful.

**Comment by f8-ptrk** [comment]
should be resolved

#antidisestablishatarianism

**Comment by jennijuju** [comment]
Thank you for creating the ticket! Lotus has been improved a lot since this ticket was created, please update your lotus to the latest release, and open a new ticket if you are still facing the issue.
